### PR TITLE
Updated ridgeback simulation with multi-robot support (MVP)

### PR DESCRIPTION
**Pull request by lilustga**

### CODE REVIEW

File: ridgeback_description/urdf/ridgeback.urdf.xacro [hunk 1]
```diff
@@ -296,6 +298,48 @@
     <origin xyz="0 0 ${deck_height}" rpy="0 0 0"/>
   </joint>
 
+  <!-- Gazebo Plugins -->
+  <gazebo>
+    <plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
+      <robotNamespace>$(arg robot_namespace)</robotNamespace>
```
Comment (seanyen) [inline ridgeback_description/urdf/ridgeback.urdf.xacro:304]: Some suggest [this](https://answers.ros.org/question/241005/gazebo-topic-publication-issue-within-namespaced-group-tag/) which adds `-gazebo_namespace /xxx/gazebo` to the `spawn_model` node can avoid to define `<robotNamespace>` in the gazebo files. Perhaps this is something to try and see if we can minimize the delta.

File: ridgeback_description/urdf/ridgeback.urdf.xacro [hunk 2]
```diff
@@ -296,6 +298,48 @@
...
+      <x_velocity_p_gain>10000.0</x_velocity_p_gain>
+      <y_velocity_p_gain>10000.0</y_velocity_p_gain>
+    </plugin>
+
+    <plugin name="imu_controller" filename="libhector_gazebo_ros_imu.so">
+      <robotNamespace>$(arg robot_namespace)</robotNamespace>
+      <updateRate>50.0</updateRate>
+      <bodyName>imu_link</bodyName>
+      <topicName>imu/data</topicName>
+      <frameId>$(arg tfpre)/base_link</frameId>
```
Comment (seanyen) [inline ridgeback_description/urdf/ridgeback.urdf.xacro:327]: it seems to me that you can do this `${prefix}/base_link`.

File: ridgeback_control/config/control.yaml [hunk 3]
```diff
@@ -19,7 +20,7 @@ ridgeback_velocity_controller:
 
   # Odometry fused with IMU is published by robot_localization, so
```
Comment (ooeygui) [inline ridgeback_control/config/control.yaml:21]: This comment isn't congruent with the setting.

File: ridgeback_control/launch/control.launch [hunk 4]
```diff
@@ -1,14 +1,25 @@
...
+ 
   <group if="$(optenv RIDGEBACK_CONTROL_EXTRAS 0)" >
     <rosparam command="load" file="$(env RIDGEBACK_CONTROL_EXTRAS_PATH)" />
   </group>
 
   <node name="controller_spawner" pkg="controller_manager" type="spawner"
         args="ridgeback_joint_publisher ridgeback_velocity_controller" />
 
-  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization" />
+  <!-- ekf localization temporarily disabled
```
Comment (ooeygui) [inline ridgeback_control/launch/control.launch:19]: What's the trigger to enable it? (in other words, it was temporarily disabled for what reason, and when will it be reenabled?)

File: ridgeback_control/launch/teleop.launch [hunk 5]
```diff
@@ -1,7 +1,10 @@
 <launch>
   <arg name="joy_dev" default="/dev/input/js0" />
   <arg name="joystick" default="true" />
-
+  <arg name="robot_namespace" default="r0"/>
+  <!-- Generate the tf_prefix from the namespace -->
+  <arg name="tfpre" default="$(arg robot_namespace)_tf" />
```
Comment (ooeygui) [inline ridgeback_control/launch/teleop.launch:6]: I think that the prefix is by convention spelled out "tf_prefix"

File: ridgeback_description/launch/description.launch [hunk 6]
```diff
@@ -1,10 +1,64 @@
...
+  <param name="tf_prefix" value="$(arg robot_namespace)_tf" />
 
+  <!-- Robot Spawn Pose -->
+  <arg name="x" default="0"/>
+  <arg name="y" default="0"/>
+  <arg name="z" default="0.1"/>
+  <arg name="yaw" default="0"/>
+
+
+<!-- "$(find ridgeback_description)/scripts/$(arg env_runner)
```
Comment (ooeygui) [inline ridgeback_description/launch/description.launch:17]: This should work - why is it disabled?

**Review by ooeygui — APPROVED** [review]
Approving. comments are non-blocking.

**Comment by seanyen** [comment]
LGTM. Merged it and we can address the comments later.

**Pull request merged**